ephypype/tests/test_import_data.py

```diff
@@ -0,0 +1,87 @@
+"""Test import_data."""
+import mne
+import os.path as op
+import numpy as np
+import scipy.io as sio
+
+import nipype.pipeline.engine as pe
+
+from ephypype.aux_tools import _change_wd
+from ephypype.nodes.import_data import ConvertDs2Fif, ImportHdf5, ImportMat
+from ephypype.import_data import _write_hdf5
+
+from numpy.testing import assert_array_almost_equal
+
+data_path = mne.datasets.testing.data_path()
+
+ds_fname = op.join(data_path, 'CTF', 'testdata_ctf.ds')
+raw_fname = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis_trunc_raw.fif')
+
+
+def test_ds2fif_node():
+    """Test ConvertDs2Fif Node."""
+
+    _change_wd()
+
+    ds2fif_node = pe.Node(interface=ConvertDs2Fif(), name='ds2fif')
+    ds2fif_node.inputs.ds_file = ds_fname
+
+    ds2fif_node.run()
+
+    raw_fif_file = ds2fif_node.result.outputs.fif_file
+    mne.io.read_raw_fif(raw_fif_file)
+
+
+def test_import_hdf5_node():
+    """Test ImportHdf5 Node."""
+
+    # write raw data in .hdf5 file
+    raw = mne.io.read_raw_fif(raw_fname)
+
+    picks = mne.pick_types(raw.info, meg=True, ref_meg=False, exclude='bads')
+    data = raw.get_data(picks=picks, start=0, stop=600)
+
+    hdf5_filename = raw_fname.replace('.fif', '.hdf5')
+    _write_hdf5(hdf5_filename, data, dataset_name='data')
```

Comment:
you don't want to usually call private functions in tests ...

Reply:
It might be worth using pytest [fixtures](https://docs.pytest.org/en/latest/fixture.html#fixtures) for data preparation steps such as `_write_hdf5` and  `_change_wd`.

Reply:
Also maybe `_write_hdf5` shouldn't be private.  If we need it here we might also want it in the workflow creation scripts which are by design available to the end user.
`_change_wd`  IMHO can be included in the testing pack as a fixture.